pkg/cvo/sync_worker.go

```diff
@@ -453,11 +459,102 @@ func (w *SyncWorker) apply(ctx context.Context, payload *updatePayload, work *Sy
 
 	setAppliedAndPending(version, total, done)
 	work.Completed++
-	reporter.Report(SyncWorkerStatus{Fraction: 1, Completed: work.Completed, Reconciling: true, VersionHash: payload.ManifestHash, Actual: update})
+	reporter.Report(SyncWorkerStatus{Fraction: 1, Completed: work.Completed, Reconciling: true, VersionHash: payloadUpdate.ManifestHash, Actual: update})
 
 	return nil
 }
 
+var (
```

Comment:
These will only reside in sync_worker until the graph iterator (pulling chunks of work from the payload graph and executing in parallel) is added.